backend/dissemination/templates/search.html

```diff
@@ -68,26 +68,26 @@ <h3>Filters</h3>
                     </div>
 
                     {% comment %} Hidden page input for use when clicking pagination buttons {% endcomment %}
-                    <input class="usa-input"
-                        aria-label="Page"
+                    <input class="usa-input audit-search-pagination-hidden"
```

Comment:
Looks like we've used some pretty verbose/specific class names elsewhere in this template, so I followed suit here.